runtime/src/iree/tooling/run_module.c

```diff
@@ -49,10 +49,14 @@ IREE_FLAG_LIST(
     "     e.g.: --output=\n"
     "  `-`: print textual form to stdout\n"
     "     e.g.: --output=-\n"
-    "  `@file.npy`: create/overwrite a numpy npy file and write buffer view\n"
+    "  `@file.npy`: create/overwrite a numpy npy file and write an ndarray\n"
     "     e.g.: --output=@file.npy\n"
-    "  `+file.npy`: create/append a numpy npy file and write buffer view\n"
+    "  `+file.npy`: create/append a numpy npy file and write an ndarray\n"
     "     e.g.: --output=+file.npy\n"
+    "  `@file.bin`: create/overwrite a binary file and write value contents\n"
+    "     e.g.: --output=@file.bin\n"
+    "  `+file.bin`: create/append a binary file and write value contents\n"
+    "     e.g.: --output=+file.bin\n"
```

Comment:
Can this be tested in https://github.com/openxla/iree/blob/main/tools/test/iree-run-module-outputs.mlir ?

We could also expand on the text in https://iree.dev/developers/general/developer-overview/#iree-run-module 
If you have ideas, the source for that is here: https://github.com/openxla/iree/blob/406626bd90a8e8e76946215dbc04efe09eed24f5/docs/website/docs/developers/general/developer-overview.md?plain=1#L121-L150

Reply:
added a test in iree-run-module-outputs.mlir